### DISCUSSION TITLE
Inclusion Xamarin.Plugin.SharedTransitions to XCT

### DISCUSSION THREAD

**Discussion by ghost** [xamarin/XamarinCommunityToolkit#1717] [Ideas]
First of all, I would like to thank the authors very much for this amazing library, which saved me a lot of trouble 😁.  Now back to my question. It would be great if you included [Xamarin.Plugin.SharedTransitions](https://github.com/GiampaoloGabba/Xamarin.Plugin.SharedTransitions) in XCToolkit, because this project seems to be dead and unusable in the future. I would really appreciate this feature. Thanks a lot.

**Comment by pictos** [comment]
Hey, @ORRNY66 thanks for the proposal. For now, we aren't accepting more features in this repo. Please open a discussion [here](https://github.com/CommunityToolkit/Maui/discussions)
